src/main/java/core/basesyntax/figures/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.Figure;
+
+public class IsoscelesTrapezoid extends Figure {
+    private static final String type = "trapezoid";
+    private final int top;
+    private final int bottom;
+    private final int height;
+
+    public IsoscelesTrapezoid(int top, int bottom, int height, String color) {
+        this.top = top;
+        this.bottom = bottom;
+        this.height = height;
+        this.color = color;
```

Comment:
While passing values directly to the constructor is not incorrect, for better readability and maintainability, consider creating variables for `top`, `bottom`, and `height` before passing them to the constructor.